src/features/events/entities/virtualEvent.entity.ts

```diff
@@ -0,0 +1,24 @@
+import { Column, Entity, OneToOne, PrimaryGeneratedColumn } from "typeorm";
+import { EventEntity } from "./event.entity";
+
+export enum EventType {
```

Comment:
please create a new directory `enums` and move the EventType enum to a new file `event-type.enum.ts` inside the enums directory